subsys/dsp/arc/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 Synopsys
```

Comment:
Since the backend is now called DSP_BACKEND_**ARCMWDT**, please update the relevant file paths to `subsys/dsp/arcmwdt/`.